roles/confluent.test/upgrade_testing/upgrade.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+
+## Variables
+
+export END_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+export KSQL_INVALID_VERSION=5.4
+
+## Change directory 
+
+cd ..
+
+## Checkout starting branch
+
+echo "Checking out $START_BRANCH branch"
+git checkout $START_BRANCH
+
+## Run Molecule Converge on scenario
+
+echo "Running molecule converge on $SCENARIO_NAME"
+molecule converge -s $SCENARIO_NAME
+
+## Checkout ending branch
+
+echo "Checkout $END_BRANCH branch"
+git checkout $END_BRANCH
+
+## Upgrade Zookeeper
+
+echo "Upgrade Zookeeper"
+(cd ../../ && ansible-playbook -i ~/.cache/molecule/confluent.test/$SCENARIO_NAME/inventory upgrade_zookeeper.yml -e kafka_broker_upgrade_start_version=$START_UPGRADE_VERSION)
```

Comment:
```suggestion
(cd ../../ && ansible-playbook -i ~/.cache/molecule/confluent.test/$SCENARIO_NAME/inventory upgrade_zookeeper.yml)
```

no need for the kafka variable i dont think

Reply:
These were the commands you ran, and have been tested to work.  I would prefer to leave this as is.

Reply:
wdym "These were the commands you ran" ?

upgrade_zookeeper.yml does not make use of the kafka_broker_upgrade_start_version variable, so it should be removed. its best to test what we document customers do: https://docs.confluent.io/current/installation/cp-ansible/ansible-upgrade.html#step-2-upgrade-zk

Reply:
These were the commands you sent me when you found the hack of where the inventory was located when tried the upgrade testing locally.  So I am literally using your commands in this instance.  If these aren't correct then I am bit concerned about the previous upgrade testing that was performed.

Reply:
Do you mean the commands here: https://github.com/confluentinc/cp-ansible-internal/pull/97 under "How Has this been Tested?

Either way, I think we can agree we should test what our documentation says right?